src/main/java/core/basesyntax/Figure.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public abstract class Figure implements FigureOperations, Info {
+    private String color;
+    private String type;
```

Comment:
`Figure` class shouldn't know anything about a type of the figure
also it would be better if we could set `color` parameter by constructor